src/components/button/Button.tsx

```diff
@@ -21,7 +21,9 @@ export interface ButtonProps {
   className?: string;
   title?: string;
   onClick?: (e: any) => void;
-
+  as?: any;
+  to?: string;
+  href?: boolean;
```

Comment:
'to' and href feel like they are duplicates of each other

Reply:
changed to linkTo i block